scss/titles.scss

```diff
@@ -0,0 +1,15 @@
+.title{ 
+height: 150px;
+margin: 40px;
+width: 420px;
+}
+.title img{
+float: left;
+height: 150px;
+}
+.title h1{
```

Comment:
Add a new line before
Check usage of html elements
Check space before curly braces
Check indentation